frontends/web/src/routes/buy/info.tsx

```diff
@@ -121,7 +121,7 @@ class BuyInfo extends Component<Props, State> {
                                     <p>
                                         {t('buy.info.disclaimer.intro.1', { name })}
                                         {' '}
-                                        (<A class={style.link} href="https://support.moonpay.com/hc/en-gb/articles/360009279877-What-are-your-supported-countries-states-and-territories-">
+                                        (<A class={style.link} href="https://shiftcrypto.support/help/en-gb/24-buy-crypto/85-what-countries-are-supported-to-buy-crypto">
```

Comment:
Make sure this host is whitelisted

Reply:
Right, thank you. I just commeted it out but kept it for later. Should I still already white list it now?

Reply:
I would do it when the time comes. Or make it a separate PR.

It would be nice if the URL in the link automatically pointed to the right language, if a translation for that language exists.

Reply:
It seems it's unclear whether we're going to use the link the disclamer at all. I would remove the whole thing for now.

Reply:
Ok updated, but I am not sure if we have to remove it now from locize the key was "buy.info.disclaimer.intro.2" cc @benma